public/.htaccess

```diff
@@ -1,5 +1,5 @@
 # Content Security Policy-Headers
 # you have to enable apache module headers to get them working
-#Header set Content-Security-Policy "default-src 'self';"
-#Header set X-Content-Security-Policy "default-src 'self';"
-#Header set X-Webkit-CSP "default-src 'self';"
+#Header set Content-Security-Policy "default-src 'self'; referrer no-referrer;"
+#Header set X-Content-Security-Policy "default-src 'self'; referrer no-referrer;"
+#Header set X-Webkit-CSP "default-src 'self'; referrer no-referrer;"
```

Comment:
Well.. this are different headers. CSP is just better supported right now where "Referrer-Policy" is another header doing the same, which is not yet supported in all browsers.